applications/common/backend/src/main/resources/libs/kyanite/common/components/table/tablecell/dialog/.content.json

```diff
@@ -18,17 +18,17 @@
         "ws:disallowedContext": ["edit:dialog"],
         "configuration": "wcm/dialogs/components/richtext/configurations/compact"
       },
-      "colspan": {
+      "columns": {
         "sling:resourceType": "wcm/dialogs/components/numberfield",
-        "name": "colspan",
+        "name": "columns",
         "min": 1,
         "max": 100,
         "label": "Columns",
```

Comment:
Now the label is the same as the name.

As for this PR, columns still translate 1:1 to colspan (and rows to rowspan) - no change here